terraform/common/variables.tf

```diff
@@ -8,4 +8,12 @@ locals {
   primary_zone_name                     = "teaching-vacancies.service.gov.uk"
   secondary_zone_name                   = "teaching-jobs.service.gov.uk"
   service_name                          = "teaching-vacancies"
```

Comment:
maybe also introduce locals for
```
arn:aws:iam::${data.aws_caller_identity.current.account_id}:user/&{aws:username}
arn:aws:iam::${data.aws_caller_identity.current.account_id}:mfa/&{aws:username}
```

Reply:
Stylistically, I don't mind `${data.aws_caller_identity.current.account_id}` and we use it in the S3 code too as a prefix (It's defined as a data source in the S3 code).